### DISCUSSION TITLE
Some kind of reselling system(for hosting companies)

### DISCUSSION THREAD

**Discussion by BlueFireRO** [pelican-dev/panel#76] [Ideas]
A system that will let users sell servers using a certain amount of resoursces,he will be able to create as many servers as he wishes to in those predefined resoursces(cpu,ram,storage) using a predefinite ip or just a predefinite port range,depending on the limitations the owner of the panel set

**Comment by ajjswift** [comment]
Giving users their own node would be great too.

**Reply by ajjswift** [reply]
Clarification:
Us hosting companies could rent a dedicated server to a user and assign it to the panel as a node, then give the user full access to resell that particular node

**Reply by BlueFireRO** [reply]
it is a way to put it,but it would be even better if you could put more resellers accounts into one node,like giving them only certain resoursces to use from the whole dedicated server,this way u could have more resellers in one dedicated server without segmenting it into separate vps for more nodes(this would help even the small companies that do not have customers to buy big dedicated servers,and also customers who want to buy smaller plans)

**Reply by ajjswift** [reply]
I think that the option for both would be good

**Comment by WolfoTheWolf** [comment]
Similar to this? https://bagou450.com/product/pterodactyl/addons/management/cloud-servers

**Reply by BlueFireRO** [reply]
yes , something similar

**Comment by ssmystic** [comment]
It would be great if it could be done like TCAdmin does their reseller accounts (allows user to create a specific number of services only and they all install in one location)

**Comment by lancepioch** [comment]
Thanks, we are actually looking into this!
